content/en/cosign/keyless.md

```diff
@@ -0,0 +1,177 @@
+---
+title: "Keyless Signatures"
+category: "Cosign"
+position: 122
+---
+
+You can review the [full design document](https://docs.google.com/document/d/1461lQUoVqbhCve7PuKNf-2_NfpjrzBG5tFohuMVTuK4/edit#) for more details.
+
+This document explains how the keyless signatures work in Cosign. This signature mode relies on the Sig store Public Good Instance.
+
+The following examples use this image:
+
+```shell
+$ IMAGE=gcr.io/dlorenc-vmtest2/demo
+$ IMAGE_DIGEST=$IMAGE@sha256:97fc222cee7991b5b061d4d4afdb5f3428fcb0c9054e1690313786befa1e4e36
+```
+
+## Usage
+
+Keyless signing:
+
+```shell
+$ COSIGN_EXPERIMENTAL=1 cosign sign $IMAGE_DIGEST
+Generating ephemeral keys...
+Retrieving signed certificate...
+Your browser will now be opened to:
+https://accounts.google.com/o/oauth2/v2/auth?access_type=offline&client_id=&redirect_uri=http%3A%2F%2F127.0.0.1%3A5556%2Fauth%2Fgoogle%2Fcallback&response_type=code
+Pushing signature to: gcr.io/dlorenc-vmtest2/demo:sha256-97fc222cee7991b5b061d4d4afdb5f3428fcb0c9054e1690313786befa1e4e36.sig
+```
+
+Keyless verifying:
+
+```shell
+$ COSIGN_EXPERIMENTAL=1 cosign verify $IMAGE
+The following checks were performed on all of these signatures:
+  - The cosign claims were validated
+  - The claims were present in the transparency log
+  - The signatures were integrated into the transparency log when the certificate was valid
+  - Any certificates were verified against the Fulcio roots.
+Certificate subject:  dlorenc@google.com
```

Comment:
Let's delete, shouldn't have someone's email here

I think this output has been updated too, can we paste in the latest output?

Reply:
I'll have to make sure I follow a keyless signature example so I can ensure I put the correct output into the docs.

Reply:
I have the updated output, but it's with `ttl.sh` which isn't particularly nice to work with as an example, as we have to push it prior to the example. So with this in mind, is it better for someone with push access to the cosign gcr to update this part? This allows us to use an official cosign image as an example as otherwise, if I was to do it, I could only use personal projects which is what we wanted to avoid here.

Reply:
See my comments above about `$IMAGE`; I think ttl.sh is fine as long as it's not in *every* example